mlir/lib/Dialect/MemRef/Transforms/NormalizeMemRefs.cpp

```diff
@@ -347,6 +358,11 @@ void NormalizeMemRefs::normalizeFuncOpMemRefs(func::FuncOp funcOp,
   for (memref::AllocOp allocOp : allocOps)
     (void)normalizeMemRef(&allocOp);
 
+  SmallVector<memref::AllocaOp> allocaOps;
```

Comment:
Can you add `AllocaOp` to the documentation of the function (also to the `normalizable...`)?